package.json

```diff
@@ -29,7 +29,30 @@
   },
   "dependencies": {
     "drizzle-orm": "0.31.4",
-    "pg": "8.12.0"
+    "pg": "8.12.0",
+    "@ai-sdk/anthropic": "^0.0.35",
+    "@ai-sdk/azure": "^0.0.17",
+    "@ai-sdk/mistral": "^0.0.26",
+    "@ai-sdk/openai": "^0.0.40",
+    "@latitude-data/compiler": "workspace:^",
+    "@latitude-data/env": "workspace:^",
+    "@t3-oss/env-core": "^0.10.1",
+    "ai": "^3.2.42",
+    "bcrypt": "^5.1.1",
+    "drizzle-kit": "^0.22.8",
+    "drizzle-orm": "0.31.4",
+    "lodash-es": "^4.17.21",
+    "uuid": "^10.0.0",
+    "zod": "^3.23.8",
+    "acorn": "^8.9.0",
+    "code-red": "^1.0.3",
+    "locate-character": "^3.0.0",
+    "yaml": "^2.4.5",
+    "dotenv": "^16.4.5",
+    "bullmq": "^5.8.3",
+    "ioredis": "^5.4.1",
+    "zod": "^3.23.8",
+    "@faker-js/faker": "^8.4.1"
```

Comment:
We treat root package.json like a monolitic Rails app `Gemfile`. I think this can work. Explained in the PR description